do500-devfile.yml

```diff
@@ -0,0 +1,94 @@
+apiVersion: 1.0.0
+metadata:
+  name: do500
+  generateName: do500-
+projects:
+  - name: todolist
+    source:
+      type: git
+      location: 'https://github.com/haithamshahin333/todolist'
```

Comment:
Is there a reason we just don't get the changes merged into `todolist` so that we can point to rht-labs? Only reason I call this out is because this seems to be a recurring theme where we just keep switching to someone elses forks :smile:

Reply:
I think it mostly comes down to: There are still changes happening in those repos hah. The change from user1/user2 etc to actual LDAP managed users has caused a number of things to need to be updated.

Reply:
@tylerauerbeck good callout - the plan is to push upstream by end of week once we stabilize some of the updates. We're making a number of changes quickly as we find small errors to get us going for next week, but once we get the chance to run through all the exercises smoothly I'll plan to push these changes back to rht-labs.